src/main/webapp/script.js

```diff
@@ -1,59 +1,15 @@
-google.charts.load("current", { packages: ["corechart"] });
-google.charts.setOnLoadCallback(makeGraph);
-
-function makeGraph() {
-  fetch("/graph")
-    .then((response) => response.json())
-    .then((data) => {
-      const hourData = new google.visualization.arrayToDataTable(data.hours);
-      const hourOptions = {
-          title: "Hours Spent per Week",
-          legend: { position: "none" },
-          vAxis: {title: "# Students"},
-          hAxis: {title: "Hours"},
-          histogram: {
-            hideBucketItems: true
-          }
-        };
-        const hourChart = new google.visualization.Histogram(document.getElementById("hours-chart"));
-        hourChart.draw(hourData, hourOptions);
-
-      const difficultyData = new google.visualization.arrayToDataTable(data.difficulty);
-      const diffOptions = {
-          title: "Diffculty of Class",
-          legend: { position: "none" },
-          vAxis: {title: "# Students"},
-          hAxis: {title: "Difficulty"},
-          histogram: {
-            hideBucketItems: true,
-          }
-        };
-        const diffChart = new google.visualization.Histogram(document.getElementById("difficulty-chart"));
-        diffChart.draw(difficultyData, diffOptions);
-    });
-}
-
-function loadBody() {
-  verifyLogin();
+function onSignIn(googleUser) {
+  const profile = googleUser.getBasicProfile();
```

Comment:
We went over this in our call earlier, but I'll just write it here in case anyone else is wondering the same thing. onSignIn essentially holds the information that you grant Google when you log in. (Like when you check the box or say "OK" to google asking you for your email, name, etc) and puts that in the profile variable.